workflow/hosts/hercules.yaml

```diff
@@ -24,8 +24,8 @@ MAKE_NSSTBUFR: 'NO'
 MAKE_ACFTBUFR: 'NO'
 DO_AWIPS: 'NO'
 SUPPORTED_RESOLUTIONS: ['C1152', 'C768', 'C384', 'C192', 'C96', 'C48']
-COMINecmwf: '/work2/noaa/global/role-global/data/external_gempak/ecmwf'
-COMINnam: '/work2/noaa/global/role-global/data/external_gempak/nam'
-COMINukmet: '/work2/noaa/global/role-global/data/external_gempak/ukmet'
-AERO_INPUTS_DIR: '/work2/noaa/global/role-global/data/gocart_emissions'
+COMINecmwf: /work/noaa/global/glopara/data/external_gempak/ecmwf
+COMINnam: /work/noaa/global/glopara/data/external_gempak/nam
+COMINukmet: /work/noaa/global/glopara/data/external_gempak/ukmet
+AERO_INPUTS_DIR: /work2/noaa/global/wkolczyn/noscrub/global-workflow/GEFS_ExtData/20250310
```

Comment:
This also doesn't seem to be correct, but I'm honestly not sure what it should be.  This directory doesn't exist.